napari/layers/image/_image_key_bindings.py

```diff
@@ -68,7 +76,8 @@ def sync_plane_normal_with_view_direction(
     return None
 
 
-@register_image_action(trans._('orient plane normal along view direction'))
+# The generator function above can't be bound to a button, so here
```

Comment:
Why not, by the way? Woudl be cool if we could just use the same function ^^' Not saying in this PR to be clear, just wondering.

Reply:
As I understand it's the way the actions work and event involved.
See:
https://github.com/napari/napari/issues/4164
https://github.com/napari/napari/pull/4265

Reply:
cool, thanks for the pointer :)